eng/tsp-core/pipelines/pr-tools.yml

```diff
@@ -64,11 +63,34 @@ extends:
                       --destination-path $(TYPESPEC_WEBSITE_BASE_PATH) \
                       --source "./website/dist/" \
                       --overwrite
+              - task: CopyFiles@2
+                displayName: "Copy VSCode extension .vsix to artifact directory"
+                inputs:
+                  SourceFolder: "$(Build.SourcesDirectory)/packages/typespec-vscode"
+                  Contents: "*.vsix"
+                  TargetFolder: "$(Build.ArtifactStagingDirectory)/vscode-extension"
+            templateContext:
+              outputs:
+                - output: pipelineArtifact
+                  path: $(Build.ArtifactStagingDirectory)/vscode-extension
+                  artifact: vscode-extension
+                  displayName: Copy VSCode extension .vsix to artifact directory
+          - job: tryit_comment
+            displayName: Create TryIt comment
+            dependsOn: tryit
+            steps:
+              - script: |
+                  echo "##[section]Retrieving typespec-vscode artifact URL..."
+                  artifactUrl=$(curl -s -u :$(System.AccessToken) "$(System.TeamFoundationCollectionUri)/$(System.TeamProject)/_apis/build/builds/$(Build.BuildId)/artifacts?artifactName=vscode-extension" | jq -r '.resource.downloadUrl')
+                  echo "Typespec-Vscode Artifact URL: $artifactUrl"
+                  echo "##vso[task.setvariable variable=vscodeUrl]$artifactUrl"
+                displayName: Get vscode artifact URL
```

Comment:
if this fails on any step (retrieving the artifact or is unable to find the url or anything) do we want to fail the whole pipeline? of is it a plan to continueOnError?

Reply:
@AlitzelMendez , if it fails on any step of the script, the whole pipeline will not fail, just not show vscode-extension in try-it comment.